### ISSUE TITLE
Deparsing stopped due to parse error | platform.pyc from Fusion 360

### ISSUE THREAD

**Issue by pastapojken** [rocky/python-uncompyle6#224]
Hope I'm not doing this totally wrong! :)
Not sure exactly what to include from `pydisasm`, so I haven't yet. Should I just run `pydisasm platform.pyc` and upload the output?

## Description

Simple, trying to decompile  platform.pyc according to [this](https://appdb.winehq.org/objectManager.php?sClass=version&iId=36468&iTestingId=104527) ( to get Fusion 360 running in wine). The "platform.pyc" file extracted from inside the .exe can be found [here](https://filebin.net/5anhyasr03k6y52g/platform.pyc?t=5bgn925u), [here](https://filebin.net/5anhyasr03k6y52g/platform.py?t=5bgn925u)'s the obtained ("failed") platform.py.

## How to Reproduce
```
uncompyle6 -o . platform.pyc 
Instruction context:
   
 228     159  LOAD_FAST                'distname'
            162  LOAD_FAST                'version'
            165  LOAD_FAST                'id'
            168  BUILD_TUPLE_3         3  ''
            171  RETURN_END_IF    
->        172_0  COME_FROM            15  '15'


# file platform.pyc
# Deparsing stopped due to parse error

# decompile failed
```
Basically, obtain exe from http://www.appstreaming.autodesk.com/install/app/73e72ada57b7480280f7a6f4a289729f/, extract it to folder, inside that folder you will find python35.zip, extract that, inside that you find aforementioned platform.pyc. 

## Expected behavior

Afaik it should just decompile it, making it possible to make changes the the line containing `maj, min, build = `, which isn't present due to the parse fail. 

## Environment
- uncompyle6 3.2.6
- Python 3.7.3
- up to date Arch Linux

**Comment by rocky** [comment]
> Hope I'm not doing this totally wrong! :)

What you are not doing right is following the instructions. When you open an issue there is a link to that.

I will not be following any elaborate step to extract bytecode. You need to do that. 

Also uncompyle6 3.3.1 is the current latest version -  so issues bug reports from that. Specifically there have been a lot of bug fixes. That said there are many more - so feel free to chip in and work on the code to address these. Failing on fixing the code, you can do also help by narrowing the problem to make it easier for those who do work on the code.

**Comment by pastapojken** [comment]
Bytecode is provided along with the truncated .py file. Will try compiling from github as Arch hasn't pushed the releases yet, hadn't noticed today's update, sorry. Should probably mention mention that I reverted to 3.2.6 due to  3.3.0 not solving my problems & being ahead of Arch repos. Will try with 3.3.1 and report back soon :)

**Comment by pastapojken** [comment]
3.3.1 fixed it, should have checked for a new release...
